Makefile

```diff
@@ -43,4 +43,10 @@ stop:
 	cat $(WORKER_PID) | xargs kill
 	cat $(FLOWER_PID) | xargs kill
 	rm *.out *.err *.log
+	kill -9 $(lsof -ti:8080)
+	rm *.out *.err
 	echo -e "\033[0;32m Airflow Stoped. \033[0m"
+
+clean:
+	docker-compose down --volumes --remove-orphans
+	docker rmi $(docker images -a -q) --force
```

Comment:
this will delete all your local images, why we would want this?

Reply:
Just to start everything from scratch, to have DAGs without runs